python/pfs/utils/optimizeBlackDots.py

```diff
@@ -0,0 +1,501 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from scipy.optimize import minimize
+from lsst.geom import AffineTransform
```

Comment:
Are we allowed to use LSST code in pfs_utils?

Reply:
@hassanxp Are you the one who should comment? If we can not use lsst.geom here, what would be an alternative? `evaluateAffineTransform` from `drp stella` takes `lsst.geom.AffineTransform` as input, and I dont see an obvious alternative?